bootstrap.sh

```diff
@@ -2,7 +2,8 @@
 
 apt-get update
 apt-get install python3 python3-pip -y
-pip3 install -r /vagrant/requirments.txt
+sudo pip3 install flask Flask-SQLAlchemy Flask-Bootstrap SQLAlchemy flask_login flask_wtf pymysql 
```

Comment:
Did you decide not to use requirments.txt?
If so, you should erase the file.

Reply:
I see this is in another PR. You should try to keep the PR changes seperate so that a change only appears in one PR and not multiple

Reply:
yah my mistake!

Reply:
i just needed the packages to make sure the code runs